akka-actor/src/main/scala/akka/dispatch/Mailbox.scala

```diff
@@ -697,6 +697,48 @@ object BoundedPriorityMailbox {
 }
 
 /**
+ * UnboundedStablePriorityMailbox is an unbounded mailbox that allows for prioritization of its contents.  Unlike the
+ * UnboundedPriorityMailbox it preserves ordering for messages of equal priority.
```

Comment:
Let's make this reference a link for easier docs navigation, like so: `[[UnboundedPriorityMailbox]]`